### PR TITLE
eslint

### PR DESCRIPTION
**Pull request by stevermeister**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,4 +57,4 @@
     "tslint": "~6.1.0",
```
Comment (bampakoa) [inline package.json:57]: We could get rid of this dependency.

File: package.json [hunk 2]
```diff
@@ -26,13 +26,24 @@
...
+    "@angular-eslint/schematics": "14.0.4",
+    "@angular-eslint/template-parser": "14.0.4",
     "@angular/cli": "^14.0.5",
     "@angular/compiler-cli": "^14.0.5",
     "@angular/language-service": "^14.0.5",
     "@types/jasmine": "~3.6.0",
     "@types/node": "^12.11.1",
+    "@typescript-eslint/eslint-plugin": "5.36.2",
+    "@typescript-eslint/parser": "5.36.2",
     "codelyzer": "^6.0.0",
```
Comment (bampakoa) [inline package.json:42]: Codelyzer was intended to be used in conjunction with TSLint. So, we can get rid of this dependency.

**Review by bampakoa — COMMENTED** [review]
LGTM! Just pointed out some packages that are not needed anymore 🙂

**Comment by stevermeister** [comment]
@bampakoa updated

**Review by bampakoa — APPROVED** [review]
🚀

**Pull request merged**